Fix for error #11801

geralexgr opened the pull request:
This commit fixes error #11801. If  fontSize text input is invalid, then
show default 12px value in textBlock.

Commit:
Fix for error #11801

This commit fixes error #11801. If  fontSize text input is invalid, then
show default 12px value in textBlock.

petetnt commented:
Hi @jerrak0s, you'll need to sign the Brackets CLA http://dev.brackets.io/brackets-contributor-license-agreement.html

jerrak0s commented:
@petetnt done. Must i do anything else?

Commit:
Fix for error #11801

his commit fixes error #11801. If  fontSize text input is invalid, then
show default 12px value in textBlock.

Commit:
commit #2

Τhis commit fixes error #11801. If  fontSize text input is invalid, then
show default 12px value in textBlock.

Commit:
commit #3

abose commented:
Why was this pr closed?

jerrak0s commented:
@abose My commit isn't working. Compile errors

abose commented:
ok. 
Raise another PR when the issue gets resolved :+1:
